src/addRemove.js

```diff
@@ -0,0 +1,80 @@
+import { addToStorage, getFromStorage } from './updateLocalStorage.js';
+
+const todoList = document.querySelector('.todo-list-items');
+const input = document.querySelector('.input-text');
+//* edit variables
+const editSection = document.querySelector('.edit__section');
+const saveEditButton = document.querySelector('.save__edit');
+const discardEditButton = document.querySelector('.discard__edit');
+const editInput = document.querySelector('.edit__input');
+
+let tasks = getFromStorage();
+
+export const addTask = () => {
+  const task = {
+    index: tasks.length + 1,
+    description: input.value,
+    completed: false,
+  };
+  tasks.push(task);
+  addToStorage(tasks);
+};
+
+//* display todo
+export const displayTasks = () => {
+  const taskData = tasks.map((item) => {
+    const { index, description } = item;
+    return `
+      <div class="todo">
+        <input type="checkbox" name="" id="" class="checkbox" />
+        <p class="description" data-edit=${index}>${description}</p>
+        <img class="dots" src="./images/Dots.png" alt="vertical-dots">;
+        <i class="fa-solid fa-file-pen edit__todo"></i>
+        <i class="fa-solid fa-trash-can remove" data-index=${index} data-info=${description}></i>
+      </div>
+    `;
+  });
+  todoList.innerHTML = taskData.join(' ');
+};
+
+//* remove and edit todo
+export const removeAndEditTodo = () => {
+  todoList.addEventListener('click', (e) => {
+    //* remove todo
+
+    if (e.target.classList.contains('remove')) {
+      e.target.parentElement.remove();
+      const taskIndex = e.target.dataset.index;
+      tasks = tasks.filter((item) => item.index !== +taskIndex);
+      tasks.forEach((item, index) => {
+        item.index = index + 1;
+      });
+      displayTasks();
+      addToStorage(tasks);
+    }
+
+    //* edit todo
+    if (e.target.classList.contains('edit__todo')) {
+      const todoDescription = e.target.parentElement.querySelector('.description');
+      editSection.classList.add('show_edit_section');
+      editInput.value = todoDescription.innerText;
+      // discard changes
```

Comment:
- [ ] Good job @LeslieAine, I just have one minor suggestion that is part of the primary requirement. When the user clicks on the checkbox the task status should be changed to ` completed: true`, when we reload the page we need to preserve this behavior and see the checkbox. Please update this for a better user experience.